CTN-001.tex

```diff
@@ -58,7 +60,7 @@
 % You can also use the \input command to include several content files.
 
 \section{Introduction}
```

Comment:
If allowed, I'd recommend changing the section name to be something like "Focal Plane Layouts" since this is the only section in the document